openpgp/keys.go

```diff
@@ -68,12 +69,12 @@ type Key struct {
 // A KeyRing provides access to public and private keys.
 type KeyRing interface {
 	// KeysById returns the set of keys that have the given key id.
-	KeysById(id uint64) []Key
+	KeysById(id uint64, fp []byte) []Key
```

Comment:
Can we document the `fp == nil` behavior here, and in the other places where the `fp` parameter is public?